pkg/maintainer/wallet/deposit_sweep.go

```diff
@@ -172,6 +159,20 @@ func FindDeposits(
 			continue
 		}
 
+		var scriptType bitcoin.ScriptType
+		depositTransaction, err := btcChain.GetTransaction(
+			event.FundingTxHash,
+		)
+		if err != nil {
+			logger.Errorf(
+				"failed to get deposit transaction data: [%v]",
+				err,
+			)
```

Comment:
We don't return on `GetTransaction` error to allow to find other deposits (deposits for which `GetTransaction` succeeded), right?

Reply:
Yep exactly. We can't return the error as it will nuke the entire process for all deposits.